tests/test_optimizely.py

```diff
@@ -5139,6 +5168,49 @@ def test_sdk_settings__accept_cache_size_and_cache_timeout(self):
         mock_logger.error.assert_not_called()
         client.close()
 
+    def test_sdk_settings__accept_default_when_cache_timeout_zero(self):
```

Comment:
Can we rename these 3 tests to something more accurate? Maybe `test_sdk_settings__accept_zero_cache_timeout`

Reply:
done